scripts/initial_setup.sh

```diff
@@ -485,6 +486,19 @@ do
     done
     SHIP_PLANS_JSON=$(echo "$SHIP_PLANS_JSON" | jq --arg nm "$CLUSTER_NAME" --arg dsh "-" --arg reg "$CLUSTER_REGION" --arg dk "$DECK_SUBDOMAIN" --arg gt "$GATE_SUBDOMAIN" --arg x509 "$X509_SUBDOMAIN" --arg vlt "$VAULT_SUBDOMAIN" --arg wd "$DOMAIN_TO_MANAGE" '. | .ship_plans += { ($nm + $dsh + $reg): { clusterPrefix: $nm, clusterRegion: $reg, deckSubdomain: $dk, gateSubdomain: $gt, x509Subdomain: $x509, vaultSubdomain: $vlt, wildcardDomain: $wd } }')
     n=$((n+1))
```

Comment:
This line needs to be removed. We only need to increment `n` once for each iteration over "ship plans". This increment will result in the wrong grafana subdomain below or some sort of index out of range error.

Reply:
that looks right. and explains why the default was showing up as grafana-sandbox instead of np

Reply:
fixed in https://github.com/homedepot/spingo/pull/62/commits/30dac5cefbfed2c8f137172bb7a2f0ec8a4a8ab7